ci-scripts/linux/local-build/linux_build.sh

```diff
@@ -0,0 +1,559 @@
+		[fr_DIRECTORY_NOT_FOUND_RETRY]="Répertoire introuvable. Réessayer. "
+		[gl_DIRECTORY_NOT_FOUND_RETRY]="Directorio non atopado. Reintentar. "
+		[pt_DIRECTORY_NOT_FOUND_RETRY]="Diretório não encontrado. Tente novamente. "
+	)
+
+    local idioma=$(echo $LANG | cut -d_ -f1)
+    local mensaje=${msg[${idioma}_$clave]:-${msg[en_$clave]}}
+
+    printf "%s" "$mensaje"
+}
+
+# ======================================================================
+# Functions
+# ======================================================================
+
+function _checkroot() {
+    # Check if the script is running WITH root privileges (and exit if true)
+    if [ "$EUID" -eq 0 ]; then
+        echo ''
+        echo "$(_msg ROOTCHECK)"
+        echo ''
+        exit 1
+    fi
+}
+
+function _checkdepends() {
+	# Check if dependencies have been installed
+	if [ ! -e "$SCRIPT_DIR/ok-packages" ]; then
+	echo""
+	echo "$(_msg DEPENDS)"
+	echo""
+	exit 1
+	fi
+}
+
+function _copyright() {
+    # Generate the backtitle with left and right alignment
+    local width=$(tput cols)  # Get the terminal width
+    local left_text="$(_msg TITLE)"
+    local right_text="(c) 2019-2025 Charlie Martínez GPLv2"
+
+    # Define a safe width to avoid text truncation
+    local safe_width=$((width - 2))  # Subtract 2 for margins
+    local max_left_length=$((safe_width / 2))  # Max length for the left text
+    local max_right_length=$((safe_width / 2)) # Max length for the right text
+
+    # Truncate the text if it exceeds the assigned space
+    if (( ${#left_text} > max_left_length )); then
+        left_text="${left_text:0:max_left_length-1}…"
+    fi
+    if (( ${#right_text} > max_right_length )); then
+        right_text="${right_text:0:max_right_length-1}…"
+    fi
+
+    # Calculate padding between the texts
+    local total_length=$(( ${#left_text} + ${#right_text} ))  # Total length of both texts
+    local padding=$((safe_width - total_length))
+
+    # If space is very small, force at least one space
+    if (( padding < 1 )); then
+        padding=1
+    fi
+
+    # Build the line with manual alignment
+    printf "%s%*s%s" "$left_text" "$padding" "" "$right_text"
+}
+   
+function _checkinternet() {
+     # Check if there's an internet connection
+    if ! ping -c 1 google.com &> /dev/null; then
+        dialog --msgbox "$(_msg INTERNET)" 10 60
+        exit 1
+    fi
+}
+
+
+function _warning() {
+    # Show warning message before compilation
+    clear
+    dialog --title "$(_msg NAVIGATE) ← →  | $(_msg CONFIRM) <Enter>" --backtitle "$(_copyright)" \
+        --yesno "$(_msg WARNING)" 10 60 >/dev/tty 2>/dev/tty
+    response=$?
+    if [ $response -ne 0 ]; then
+       	# If the user selects 'No', exit the program
+       	clear
+       	exit 1
+    fi
+}
+
+function _cloningTahoma() {
+    local TAHOMA_DIR="./tahoma2d"
+    
+    # Check if the folder exists in the default location
+    if [ -d "$TAHOMA_DIR" ]; then
```

Comment:
When I run this script like this:

1. `git clone https://github.com/tahoma2d/tahoma2d`
2. `cd tahoma2d/ci-scripts/linux/local-build`
3. `chmod +x ./linux_*`
4. `./linux_depends.sh`
5. `./linux_build.sh`

I get an error about not being able to find Tahoma. This seems like it should be a supported use-case.